pkg/provider/containergroup_to_pod.go

```diff
@@ -131,7 +132,7 @@ func aciContainerStateToContainerState(cs *azaciv2.ContainerState) v1.ContainerS
 			},
 		}
 	// Handle the case of completion.
-	case "Succeeded":
+	case "Succeeded", "Terminated":
```

Comment:
The return should have a Reason other than "Completed" that goes with the "Terminated" state.

Reply:
Usually, CRIs set Reason to Completed when state is terminated: 
https://github.com/Mirantis/cri-dockerd/blob/39a4ff0619d49651055509a0d2c3895f33772329/core/container_status.go#LL92C15-L92C24
https://github.com/containerd/containerd/blob/4b7145cfd38d76bc763e3cee1210963bc6c22680/pkg/cri/server/container_status.go#LL91C31-L91C31

Maybe Terminated as another meaning in ACI ? What reason should it be ?